libraries/src/test/java/com/baeldung/circularfifoqueue/example/CircularFifoQueueTest.java

```diff
@@ -0,0 +1,161 @@
+package com.baeldung.circularfifoqueue.example;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.collections4.queue.CircularFifoQueue;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CircularFifoQueueTest {
+
+	private static final int DEFAULT_SIZE = 32;
+	
+	private static final int FIXED_SIZE = 5;
+	
+	private static final int COLLECTION_SIZE = 7;
+	
+	private static final String TEST_COLOR = "Red";
+	
+	private static final String TEST_COLOR_BY_INDEX = "Blue";
+
+	@Test
+	public void whenUsingDefualtConstructor_correctSizeQueue() {
+		CircularFifoQueue bits = new CircularFifoQueue();
+
+		Assert.assertEquals(DEFAULT_SIZE, bits.maxSize());
+	}
+
+	@Test
+	public void givenAddElements_whenUsingIntConstructor_correctSizeQueue() {
+		CircularFifoQueue colors = new CircularFifoQueue(5);
```

Comment:
Use CircularFifoQueue<String> when declaring, and use the diamond operator when constructing. For example: new CircularFifoQueue<>()